tests/test_search.py

```diff
@@ -1526,6 +1532,27 @@ def test_profile_limited(client):
     assert len(res.docs) == 3  # check also the search result
 
 
+@pytest.mark.redismod
+def test_vector_field(modclient):
+    modclient.flushdb()
+    modclient.ft().create_index(
+        (
+            VectorField(
+                "v", "HNSW", ["TYPE", "FLOAT32", "DIM", 2, "DISTANCE_METRIC", "L2"]
```

Comment:
@chayim Maybe it's better to pass the attributes as a dictionary?

Reply:
It feels a bit clunky. I'd like to two do things before we merge:

1. Try the dictionary approach (and document the kwargs)
2. Add a jupyter notebook example. 

WDYT @Avital-Fine ?